deploy/crds/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -684,6 +684,11 @@ spec:
                         description: Remote Storage Volume name
                         type: string
                     type: object
+                  maxConcurrentAppDownloads:
+                    description: Maximum number of apps that can be downloaded at
```

Comment:
Am I right in saying the `maxConcurrentAppDownloads` will be 1 global config which is applicable  for all App sources of the CR?

Reply:
Yes